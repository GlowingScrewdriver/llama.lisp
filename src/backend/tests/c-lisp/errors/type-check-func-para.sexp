(c-lisp
    (define ((print int) (n int)))

    (define ((func_print void) (n int) (m float)) 
        (call print n)
        (call print m))

    (define ((main void)) 
        (declare k int)
        (declare j float)
        (set k 50)
        (set j 10.0)
        (call func_print k j)))