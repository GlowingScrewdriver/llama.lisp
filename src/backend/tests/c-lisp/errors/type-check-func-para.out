Error in statement: ['call', 'print', 'm']
__main__.CodegenError: type mismatch, declared function parameter type is int
